Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -355,10 +355,11 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
-- Note: When comparing two numeric values within a specified accuracy, i.e. the
-analogue of [`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal)
-, please import [the swift-numerics package](https://github.com/apple/swift-numerics)
-and use the ``isApproximatelyEqual()`` function.
+- Note: To check that two numeric values are equivalent within a specified
+accuracy, as in the analogue of 
```

Comment:
@chuckdude @medreisbach Does Apple style prefer "analogue" or "analog"?

Reply:
/me places his bet on "analog" - the American english spelling

Reply:
I suspect you're right, hence why I asked! 😃